app/views/teams/_form.html.erb

```diff
@@ -11,22 +11,61 @@
     </div>
   <% end %>
 
+  <input type="hidden" id="department" value='<%= @department.id %>'>
   <div class="form-group">
     <%= form.label :name %><br>
     <%= form.text_field :name, required: true, class: "form-control" %>
   </div>
+  
   <div class="form-group">
-    <%= form.label :department_id, "Department" %><br>
-    <%= form.collection_select :department_id, current_employee.company.departments.order(:name),:id,:name, include_blank: true, class: "form-control" %>
+    <%= form.label :team_lead_id, "Team Leader" %>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
use css classes ( margin-left , margin-right, margin-top, margin-bottoms ) instead of & nbsp